cpp/CMakeLists.txt

```diff
@@ -174,127 +174,312 @@ set(CUGRAPH_SOURCES
     src/detail/collect_local_vertex_values.cu
     src/detail/groupby_and_count.cu
     src/detail/collect_comm_wrapper.cu
```

Comment:
Similarly, this file is MG only but explicitly instantiates based on vertex_t, edge_t, and weight_t. May create mg32, mg3264, and mg64?

Reply:
Already working on it

Reply:
Done.